cmd/flag/open_stream_flags.go

```diff
@@ -17,15 +17,32 @@ package flag
 
 import (
 	"github.com/spf13/cobra"
+	"time"
+)
+
+var (
+	defaultEnableAutoClose = false
+	defaultAutoCloseDelay  = 5 * time.Second
+	defaultAutoCloseTime   = "2006-01-02'T'15:04:05Z"
 )
 
 type OpenStreamFlag struct {
-	StreamId string
+	EnableAutoClose bool
+	AutoCloseDelay  time.Duration
+	AutoCloseTime   string
+	StreamId        string
 }
 
 // Add a flag to the command.
 func (f *OpenStreamFlag) AddFlags(cmd *cobra.Command) {
 	cmd.Flags().StringVarP(&f.StreamId, "stream-id", "r", "", "The StreamId to resume.")
+	cmd.Flags().BoolVarP(&f.EnableAutoClose, "enable-auto-close", "", defaultEnableAutoClose,
+		"When set to true, the stream will close after a specified auto close time.")
+	cmd.Flags().DurationVarP(&f.AutoCloseDelay, "auto-close-delay", "", defaultAutoCloseDelay,
+		"The duration to wait before ending the stream with no more data incoming.")
+	cmd.Flags().StringVarP(&f.AutoCloseTime, "auto-close-time", "", defaultAutoCloseTime,
```

Comment:
However we end up arranging the params, I think it would be a good idea to show the format of the timestamp-- something like `YYYY-MM-DDThh:mm:ssZ`, or mentioning it is an ISO 8601 datetime. 

One last thing about timestamps-- we should confirm that the accepted format here is the same as timestamps used in other commands (listing passes, unavailability windows, etc).

Reply:
Added some help text explaining the date format.